docs/asset-store/11-02-asset-metadata-service.md

```diff
@@ -12,4 +12,18 @@ This table shows the Asset Metadata Service custom metrics, their types, and des
 
 Apart from the custom metrics, the Asset Metadata Service also exposes default Prometheus metrics for [Go applications](https://prometheus.io/docs/guides/go-application/).
 
+To see a complete list of the custom and Go metrics, run this command:
```

Comment:
Hm... I think something generic like 
```suggestion
To see a complete list of metrics, run this command:
```

It will be better, also in other places. For me, we don't need to mention that here, this is just an endpoint with all metrics for such microservice.